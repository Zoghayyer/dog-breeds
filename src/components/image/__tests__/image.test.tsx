import React from 'react';
import { shallow } from 'enzyme';
import Image, { ImageProps } from '../image';

describe('(Component) Image', () => {
  let component;
  const defaultProps: ImageProps = {
    alt: 'My fake alt text',
    className: '',
    src: 'src',
  };

  const shallowImage = (extendedProps: Partial<ImageProps> = {}) => {
    component = shallow(<Image {...defaultProps} {...extendedProps} />);
  };

  beforeEach(() => shallowImage());

  it('renders without crashing', () => {
    shallow(<Image {...defaultProps} />);
    expect(component).toMatchSnapshot();
  });
});
