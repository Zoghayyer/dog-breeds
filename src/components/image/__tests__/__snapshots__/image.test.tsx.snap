// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Image renders without crashing 1`] = `
<img
  alt="My fake alt text"
  className=""
  src="src"
/>
`;
